server/src/test/java/org/elasticsearch/index/query/MatchQueryBuilderTests.java

```diff
@@ -300,6 +301,14 @@ public void testFuzzinessOnNonStringField() throws Exception {
         query.toQuery(context); // no exception
     }
 
+    public void testFuzziness() throws Exception {
+        MatchQueryBuilder query = new MatchQueryBuilder(STRING_FIELD_NAME, "this is a testt");
+        query.fuzziness(Fuzziness.TWO);
+        QueryShardContext context = createShardContext();
+        Query q = query.toQuery(context);
+        System.out.println(q);
```

Comment:
I suspect you didn't really want to add a test and only added this test to see what the behavior was?

Reply:
Oops, yes...